Run make config-rebuild to update config/docs

jdavcs opened the pull request:
This is to update galaxy.yml.sample and galaxy_options.rst to be consistent with the latest update to config_schema.yml (PRs https://github.com/galaxyproject/galaxy/pull/7897 and https://github.com/galaxyproject/galaxy/pull/8118)

(generated with `make config-rebuild`)

NOTE: I assume we run this each time we update config_schema.yml? I saw this while rebuilding with my own updates, and I didn't want to add this older update to an unrelated PR.

jdavcs committed:
Run make config-rebuild to update config/docs

This is to update galaxy.yml.sample and galaxy_options.rst to be
consistent with the latest update to config_schema.yml (#5651edc)

(generated with make config-rebuild)

hexylena reviewed (approved):
> NOTE: I assume we run this each time we update config_schema.yml? I

Correct, not sure where the documentation is for that, perhaps we should add a comment to config_schema.yaml

jdavcs committed:
Add reminder to run make config-rebuild

In reference to:
https://github.com/galaxyproject/galaxy/pull/8220#pullrequestreview-253772398

jdavcs commented:
Thanks @erasche . And yes, added a comment to config_schema.yml.
